wgengine/packet/packet.go

```diff
@@ -86,8 +74,11 @@ type QDecode struct {
 }
 
 func (q *QDecode) String() string {
-	if q.IPProto == Junk {
-		return "Junk{}"
+	switch q.IPProto {
+	case IPv6:
+		return "IPv6{???}"
+	case Junk:
+		return "Junk{???}"
```

Comment:
Since this'll show up in user logs, I'd rather not call their traffic Junk. How about `Other` or `Unclassified` or `Unknown`?